### PR TITLE
finish_mobile_version

### PR DESCRIPTION
**Pull request by harshita02**
It is the complete mobile version of the Portfolio webpage with the addition of the "works" and "about" section, the works include CSS Grid and Flexbox. "ul" and "li" tags are used for the categories of both the sections.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -69,3 +78,222 @@ body {
...
+  width: 288px;
+  height: 126px;
+  margin-top: 20px;
+  margin-left: 16px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  color: #1a2236;
+}
```
Comment (Bondok6) [inline style.css]: - [ ] According to the requirements and the original design, You must stick to the Figma design as much as possible, So I suggest to align this paragraph at left to match your Figma design 👍

> You can add `text-align: left` to fix this 💡
> Or Remove the `<center>` HTML tag 💡

| Your Design  | Figma Design  |   
| --- | --- |
|   |  |

File: style.css [hunk 2]
```diff
@@ -69,3 +78,222 @@ body {
...
+  width: 289px;
+  margin-left: 16px;
+  margin-top: 15px;
+  background-color: #febf52;
+}
+
+button:hover {
+  background-color: yellowgreen;
+  cursor: pointer;
+}
```
Comment (Bondok6) [inline style.css]: - [ ] Great job Implementing the button interactions (hover) 💯 However, إou have to use the same colors as in your Figma design 👍

image

File: index.html [hunk 3]
```diff
@@ -22,10 +22,155 @@ <h1 class="profileName">Harshita Jain</h1>
...
+            <img src="images/ic_download.svg">
+        </button>
+        <div class="languages">
+            <h4 class="set-h">Languages</h4>
+            <ul class="set-ul">
+                <li>JavaScript(ES5/ES6)</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+            </ul>
```
Comment (Bondok6) [inline index.html:149]: - [ ] According to the requirements and the original design, You must stick to the Figma design as much as possible(e.g., fontsm paddings, margins, ..), So please work on this section 👍

| Your Design  | Figma Design  |   
| --- | --- |
|   |  |


> I suggest removing the `<center>` HTML tag 💡

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi @harshita02 🙋‍♂️, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No Linter errors ✅
- Correct GitHub flow ✅
- Your repo is professional ✅
- Great job using semantic HTML5 tags ✅


## Required Changes ♻️
- [ ] _Check the comments under the review._ ⬇

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -22,10 +22,155 @@ <h1 class="profileName">Harshita Jain</h1>
...
+                <li>Bootstrap</li>
+                <li>Ruby on rails</li>
+                <li>RSpec</li>
+                <li>Capibara</li>
+                <li>Selenium</li>
+            </ul>
+            <img class="set-img" src="images/Group 18.png">
+        </div>
+        <div class="skills">
+            <h4 class="set-h">Skills</h4>
```
Comment (algerina) [inline index.html:164]: - [ Optional] Kindly align this h4 to the left to match the other sections and the Figma design.

**Review by algerina — APPROVED** [review]
Hi @harshita02 , 
Good job so far! 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSEs
 



 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**